src/content/app/tools/blast/state/blast-form/blastFormSlice.ts

```diff
@@ -54,6 +58,8 @@ const initialBlastFormSettings: BlastFormSettings = {
   jobName: '',
   sequenceType: 'dna',
   sequenceSelectionMode: 'automatic',
+  databaseSelectionMode: 'automatic',
+  programSelectionMode: 'automatic',
```

Comment:
I don't understand the purpose of this flag. The program is always a computed variable derived from a combination of input sequence type and database sequence type. Why does this flag need to exist?